local-tests/run_nodes.py

```diff
@@ -0,0 +1,36 @@
+#!/bin/env python
+
+# Short script demonstrating the basic usage of `chainrunner` package.
+# Reproduces (more or less) the behavior of `run_nodes.sh`.
+# For running local experiments it's much more convenient to manage the chain
+# using an interactive environment (Python console, Jupyter notebook etc.)
+
+from time import sleep
+from chainrunner import Chain, Seq, Generate_keys
+
+nodes = 4
+workdir = '.'
+binary = '../target/release/aleph-node'
+
+phrases = ['//Alice','//Bob','//Cedric','//Dick', '//Ezekiel', '//Fanny', '//George', '//Hugo']
+keys_dict = Generate_keys(binary, phrases)
+keys = list(keys_dict.values())
+
+chain = Chain(workdir)
+
+print(f'Bootstraping chain for {nodes} nodes')
+chain.bootstrap(binary, keys[:nodes], chain_type='local', millisecs_per_block=2000, session_period=40)
```

Comment:
just to avoid confusion, check whether `nodes <= len(keys)`

Reply:
Done